### ISSUE TITLE
Comment what the key value pairs here are 

### ISSUE THREAD

**Issue by thesimplekid** [cashubtc/cdk#373]
https://github.com/cashubtc/cdk/blob/b23b48c3fe18a3d940c8b8042b1e08dade77671a/crates/cdk/src/cdk_database/mint_memory.rs#L22-L31

**Comment by thesimplekid** [comment]
https://github.com/cashubtc/cdk/blob/b23b48c3fe18a3d940c8b8042b1e08dade77671a/crates/cdk-redb/src/mint/mod.rs#L28-L42

**Comment by mubarak23** [comment]
@thesimplekid if you can tag this as Good first issue, and assign me

also add more context

**Comment by thesimplekid** [comment]
I don't think this is a good first issue. There are some other database clean ups that I want to do and will include this in that.
